src/components/ConversationSettings/ConversationSettingsDialog.vue

```diff
@@ -181,7 +176,7 @@ export default {
 		},
 
 		showMediaSettingsToggle() {
-			return (!supportFederationV1 || !this.conversation.remoteServer)
+			return (!hasTalkFeature(this.token, 'federation-v1') || !this.conversation.remoteServer)
```

Comment:
This is not dynamic

Reply:
Component is always mounted, and we can call it from any route (LeftSidebar -> Conversation -> Conversation Settings), so we should rely on token of this component